storage/local/chunk/chunk.go

```diff
@@ -57,96 +66,96 @@ func (ce *ChunkEncoding) Set(s string) error {
 
 const (
 	// Delta encoding
-	Delta ChunkEncoding = iota
+	Delta Encoding = iota
 	// DoubleDelta encoding
 	DoubleDelta
 	// Varbit encoding
 	Varbit
 )
 
-// ChunkDesc contains meta-data for a chunk. Pay special attention to the
+// Desc contains meta-data for a chunk. Pay special attention to the
 // documented requirements for calling its methods concurrently (WRT pinning and
 // locking). The doc comments spell out the requirements for each method, but
 // here is an overview and general explanation:
 //
 // Everything that changes the pinning of the underlying chunk or deals with its
-// eviction is protected by a mutex. This affects the following methods: pin,
-// unpin, refCount, isEvicted, maybeEvict. These methods can be called at any
+// eviction is protected by a mutex. This affects the following methods: Pin,
+// Unpin, RefCount, IsEvicted, MaybeEvict. These methods can be called at any
 // time without further prerequisites.
 //
 // Another group of methods acts on (or sets) the underlying chunk. These
 // methods involve no locking. They may only be called if the caller has pinned
 // the chunk (to guarantee the chunk is not evicted concurrently). Also, the
 // caller must make sure nobody else will call these methods concurrently,
-// either by holding the sole reference to the chunkDesc (usually during loading
-// or creation) or by locking the fingerprint of the series the chunkDesc
-// belongs to. The affected methods are: add, maybePopulateLastTime, setChunk.
+// either by holding the sole reference to the ChunkDesc (usually during loading
```

Comment:
Oops, I thought I had renamed it in the comments everywhere too, but something must have gone wrong. Fixed.